docs/ref/django-admin.txt

```diff
@@ -1614,6 +1618,9 @@ Example usage::
 
 .. django-admin:: createsuperuser
 
+.. envvar:: DJANGO_SUPERUSER_EMAIL
```

Comment:
`DJANGO_SUPERUSER_EMAIL` is just an example, it can be misleading that we added `envvar` for it and not for other fields. I will revert this.